src/librustdoc/html/render/mod.rs

```diff
@@ -1527,10 +1513,14 @@ fn render_impl(
             );
         }
     }
-    let details_str = if impl_items.is_empty() {
-        ""
-    } else {
-        "<details class=\"rustdoc-toggle implementors-toggle\" open><summary>"
+    let toggled = impl_items.is_empty() && default_impl_items.is_empty();
+    let open_details = |close_tags: &mut String| {
+        if !toggled {
```

Comment:
The logic of this variable name is backwards - `toggled` should mean "wrapped in a details tag" and `!toggled` should mean "not wrapped in a details tag".

Reply:
Oups sorry, completely inverted it. Fixed!